src/ncp/ncp_spinel.hpp

```diff
@@ -124,19 +201,41 @@ class NcpSpinel
     void HandleNotification(const uint8_t *aFrame, uint16_t aLength);
     void HandleResponse(spinel_tid_t aTid, const uint8_t *aFrame, uint16_t aLength);
     void HandleValueIs(spinel_prop_key_t aKey, const uint8_t *aBuffer, uint16_t aLength);
+    void HandleResponseForCommand(spinel_tid_t aTid, otError aError);
 
     spinel_tid_t GetNextTid(void);
     void         FreeTid(spinel_tid_t tid) { mCmdTidsInUse &= ~(1 << tid); }
 
+    using EncodingFunc = std::function<otError(void)>;
+    otError SetProperty(spinel_prop_key_t aKey, const EncodingFunc &aEncodingFunc);
+    otError SendEncodedFrame(void);
+
+    void GetFlagsFromSecurityPolicy(const otSecurityPolicy *aSecurityPolicy, uint8_t *aFlags, uint8_t aFlagsLength);
+
+    otError EncodeDatasetSetActiveTlvs(const otOperationalDatasetTlvs &aActiveOpDatasetTlvs);
+
     ot::Spinel::SpinelDriver *mSpinelDriver;
     uint16_t                  mCmdTidsInUse;              ///< Used transaction ids.
     spinel_tid_t              mCmdNextTid;                ///< Next available transaction id.
     spinel_prop_key_t         mWaitingKeyTable[kMaxTids]; ///< The property keys of ongoing transactions.
+    spinel_command_t mCmdTable[kMaxTids]; ///< The mapping of spinel command and tids when the response is LAST_STATUS.
```

Comment:
I wonder why this line is not aligned with others? Doesn't the formatter apply here?

Reply:
Yeah that's interesting and annoying. I had a few tries and it seems the pretty tries to make it < 120 word by compacting the line instead of starting a new line. So I manually start a new line for the comment.